__tests__/app.test.js

```diff
@@ -36,7 +36,52 @@ describe('get:', () => {
 				});
 		});
 		test('status:404 Not Found', () => {
-            return request(app).get('/api/topic').expect(404).then(res => {expect(res.error.text).toBe("error: path not found")})
+			return request(app)
+				.get('/api/topic')
+				.expect(404)
+				.then(res => {
+					expect(res.error.text).toBe('error: path or content not found');
+				});
+		});
+	});
+	describe('/api/articles/:article_id', () => {
+		test('status:200 returns a single object', () => {
+			return request(app)
+				.get('/api/articles/1')
+				.expect(200)
+                .then(res => {
+					expect(res.body).toBeInstanceOf(Object);
+				});
 		});
+		test('status:200 returns a object with the required properties', () => {
+			return request(app)
+				.get('/api/articles/1')
+				.expect(200)
+				.then(res => {
+					expect(res._body).toEqual(
+						expect.objectContaining({
+							author: expect.any(String),
+							title: expect.any(String),
+							article_id: expect.any(Number),
+							body: expect.any(String),
+							topic: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+						}),
```

Comment:
You can be more specific here! If you're going to articles/1 then you know which article you are getting back. article_id would be 1 etc.